### PR TITLE
fix(docs): Documentation Grammatical Errors.

### PR DESCRIPTION
**Pull request by gpu004**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.
- fixed some small grammatical errors

Closes #XXXXX

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ Our community also has:
 
 - A [forum](https://forum.freecodecamp.org) where you can usually get programming help or project feedback within hours.
 - A [YouTube channel](https://youtube.com/freecodecamp) with free courses on Python, SQL, Android, and a wide variety of other technologies.
-- A [technical publication](https://www.freecodecamp.org/news) with thousands of programming tutorials and articles about math and computer science.
+- A [technical publication](https://www.freecodecamp.org/news) with thousands of programming tutorials and articles about maths and computer science.
```
Comment (Sboonny) [inline README.md:164]:
```suggestion
- A [technical publication](https://www.freecodecamp.org/news) with thousands of programming tutorials and articles about mathematics and computer science.
```

**Review by Sboonny — COMMENTED** [review]
According to https://www.grammar.com/math_vs._maths, both math and maths are correct, so I am wondering if it's better to change it to mathematics, to avoid unnecessary pull request in the future

**Comment by ShaunSHamilton** [comment]
Thank you, for opening this pull request.

This is a standard message notifying you that we've reviewed your pull request and have decided not to merge it. We would welcome future pull requests from you. Please take a minute to read our [contributors docs](https://contribute.freecodecamp.org/#/index)

Thank you, and happy coding.

**Comment by ShaunSHamilton** [comment]
After my eyes adjusting, I see the suggestions you have made are correct. Sorry, I looked at the diff wrong.

File: docs/FAQ.md [hunk 2]
```diff
@@ -81,11 +81,11 @@ community, and live by our [code of conduct](https://www.freecodecamp.org/news/c
 Here are some recommended paths for some of our platforms:
 
 - To be a **Discord/Chat** moderator, have an active presence in our chat and
-  have positive engagements with others, while also learning and practicing how
+  have positive engagements with others, while also learning and practising how
```
Comment (naomi-lgbt) [inline docs/FAQ.md:84]:
```suggestion
  have positive engagements with others, while also learning and practicing how
```

As we are an American-based organisation, we do use American spellings.

Comment (ShaunSHamilton) [inline docs/FAQ.md:84]: In keeping with American English, this should not change

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
I'm in agreement with Sboony's suggestion above, and have one of my own.

Would be happy to merge once these two are resolved. If you head to the `files` tab, you can view our suggestions there and hit `Add suggestion to batch`. Once done, you will see a `commit changes` button that allows you to create a commit with both of our suggestions.

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @tusharxoxoxo, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Review by naomi-lgbt — APPROVED** [review]

**Comment by naomi-lgbt** [comment]
Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**